lib/far_mar/base.rb

```diff
@@ -0,0 +1,22 @@
+class FarMar::Base
+  attr_accessor :attributes
+  FILE = ("csv.csv")
+
+  def initialize
+    @attributes = :attributes
+  end
+
+  def self.all
+    all_self = CSV.read(self::FILE)
+    all_self.collect do |attrs|
+    self.new(attrs)
```

Comment:
This line should be indented because it's between a `do` and `end`. Or you could put it on a single line:

```ruby
all_self.collect { |attrs| new(attrs) }
```